components/Card/card.css

```diff
@@ -0,0 +1,116 @@
+:root{
+    --item-name-color:#535766;
+    --box-shadow-card:rgba(0, 0, 0, 0.24);
+    --discount:red;
+    --badge-background: rgb(124, 219, 80);
+    --font-color:whitesmoke;
+}
+.component-card{
+    display: flex;
+}
+.card{
+    position: relative;
+    width:fit-content;
+    box-shadow: var(--box-shadow-card) 0px 3px 8px;
+    padding: 1rem;
+    margin: 1rem;
+    font-size:1rem;
+    font-weight:400;
+}
+.wishlist{
+    display: none;
+}
+.card:hover > .wishlist {
+    display: block;
+    margin:1rem 0 0 1rem;
+    background-color: inherit;
+    border:1px solid black;
+    width:fit-content;
+    width:10rem;
+    align-self: center;
+}
+
+.card:hover>.Item{
```

Comment:
On hover, property changes should be done, keeping in mind, the experience of mobile users. 
Mobile doesn't have any option of hover.